### PR TITLE
Expand docstring and setup autoapi

### PR DESCRIPTION
**Pull request by humitos**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -40,18 +40,14 @@
...
 extensions = [
     'notfound.extension',
-    # 'autoapi.extension',
+    'autoapi.extension',
     'sphinx_tabs.tabs',
     'sphinx-prompt',
     'sphinxemoji.sphinxemoji',
 ]
 
-notfound_context = {
```
Comment (stsewd) [inline docs/conf.py]: We are using the default context now, right?

File: docs/conf.py [hunk 2]
```diff
@@ -24,9 +24,9 @@
 author = 'Manuel Kaufmann'
 
 # The short X.Y version
-version = ''
 # The full version, including alpha/beta/rc tags
-release = '0.2.1'
+import notfound
+version = release = notfound.version
```
Comment (stsewd) [inline docs/conf.py]: nip: I prefer this in two lines

```python
version = notfound.version
release = version
```

File: docs/conf.py (hunk 1 above)
Comment (humitos) [reply docs/conf.py]: Yes.

**Comment by humitos** [comment]
Baby steps. We are getting there, and even further.

**Pull request merged**